app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<div class="users">
+  <h1>Users</h1>
+  <% @users.each do |user| %>
+    <%= link_to "/users/#{user.id}" do %>
+    <div class="user-info-wrapper">
+      <%= image_tag user.photo, :class => 'user-avatar' %>
+      <div class="user-details">
+        <p><%= user.name %></p>
+        <span>Number of posts: <%= user.post_counter %></span>
```

Comment:
- [ ] Well done on your progress! As the previous reviewer mentioned, you've made significant improvements to your UI. However, there's a small adjustment needed to align with the wireframe specifications. In the wireframe, the "post counter" span should be right-aligned, while in your current code, it appears to be left-aligned. To ensure consistency with the design, I recommend aligning the "post counter" span to the right. This subtle adjustment will bring your UI in line with the wireframe and enhance its visual appeal. Keep up the great work! 🚀🎨

| Your | Design |
| --- | --- |
|  ![image](https://github.com/simplegoose/blog_app/assets/55840999/a4759012-8b71-4dff-ab3b-4e8d4c27413c)| ![image](https://github.com/simplegoose/blog_app/assets/55840999/c8ebc656-42e6-473c-8306-73678edf7872)|